modules/openapi-generator/src/main/resources/cpp-qt5-client/CMakeLists.txt.mustache

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(client)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -std=c++14 -Wall -Wno-unused-variable")
+
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Network REQUIRED)
+
+file(GLOB SRCS
+    ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+)
+
+include_directories(
+    ${Qt5Core_INCLUDE_DIRS}
+    ${Qt5Network_INCLUDE_DIRS}
+)
```

Comment:
can be removed (take a look at my comments from cpp-qt5/cMakeLists.txt)

Reply:
done